epictrack-web/src/components/staff/StaffList.tsx

```diff
@@ -106,6 +106,12 @@ const StaffList = () => {
         },
         filterSelectOptions: positions,
         filterFn: "multiSelectFilter",
+        sortingFn: (rowA, rowB) => {
+          const positionA = rowA.original.position?.sort_order;
+          const positionB = rowB.original.position?.sort_order;

```

Comment:
I don't think this ticket is about sorting the table based on position. I think this ticket is for sorting the position filter based on hierarchy.

Reply:
Fixed, please recheck